.github/workflows/main.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+  pull:
+  schedule:
+    - cron: '15 3 * * *'
```

Comment:
:+1:

Reply:
if you would like to be super picky we can exclude sat and sun
'15 3 * * 1,2,3,4,5' i think this is the syntax